source/common/network/address_impl.cc

```diff
@@ -300,8 +357,40 @@ PipeInstance::PipeInstance(const std::string& pipe_path, mode_t mode,
   pipe_.mode_ = mode;
 }
 
+PipeInstance::PipeInstance(absl::Status& error, const sockaddr_un* address, socklen_t ss_len,
```

Comment:
I'm not spotting where this new constructor variant is used, can you point out?

Reply:
Address::InstanceFactory::createInstancePtr<Address::PipeInstance>(sun, ss_len); In addressFromSockAddr()